utils/tolerance.stanza

```diff
@@ -0,0 +1,60 @@
+#use-added-syntax(jitx)
+defpackage ocdb/tolerance:
```

Comment:
should we create unit tests for the packages in this repo?

Reply:
Done - also good call, it found a few bugs